docs/platforms/javascript/common/session-replay/issue-types.mdx

```diff
@@ -0,0 +1,50 @@
+
+You can configure which issue types are created by visiting Project Settings > Replay and toggling detection on or off for each issue type.
+
+![Replay settings page](./img/replay-settings.png)
+
+## Configuring Rage Click Issues
+
+"Slow clicks" (also called "dead clicks") are only detected on `<button>`, `<input>`, and `<a>` elements that don't lead to updates to the DOM or a page scroll within 7 seconds. When the user clicks on one of these elements 3 or more times within that 7 second time frame, it indicates frustration, and the SDK registers a ["rage click"](/product/issues/issue-details/replay-issues/rage-clicks/).
+
+Sometimes rage or dead clicks will be detected on elements which are not expected to mutate the DOM, like a "Print" or "Download" button. In this case, you can configure `slowClickIgnoreSelectors`, which will prevent those buttons from generating new issues. See <PlatformLink to="/session-replay/troubleshooting/#why-am-i-seeing-too-many-or-too-few-rage-clicks">Why am I seeing too many or too few rage clicks?</PlatformLink> for more.
+
+## Configuring Hydration Errors
+
+When a [hydration error](/product/issues/issue-details/replay-issues/hydration-error/) occurs in your React app, the SDK will emit an error object as well as a replay breadcrumb, both of which represent the problem. Both of these are sent to the server for processing.
+
+The error event, before considering available quota, is subject to [Inbound Filters](https://docs.sentry.io/concepts/data-management/filtering/) and may be dropped. If the inbound filters are disabled, the error is accepted (consuming error quota) and an issue is created. Error events may or may not have an attached replay to help debug the problem.
+
+The replay breadcrumb is stored as part of the replay session if the server has already accepted the replay. While storing the breadcrumb, the ingest pipeline looks at your project settings and will create a replay issue based on this hydration breadcrumb if the preference is enabled in Project Settings > Replay. Each event instance inside the issue will have an associated replay, to make debugging easier.
+
+You can have your preferences configured in one of four ways:
+| Inbound Filters | Replay Hydration Error toggle | Result |
+| --- | --- | --- |
+| enabled | disabled | No issues will be created. |
+| enabled | enabled | Issue created based on replay breadcrumb data. |
+| disabled | disabled | Issue created from captured error data. |
+| disabled | enabled | Two issues created from both replay breadcrumb, and error data. The issues are duplicates. |
+
+Enable both preferences for the best experience. This will result in one Issue inside your issue stream where each event has an associated replay to help debug the root cause.
```

Comment:
```suggestion
Enable both preferences for the best experience. This will result in one issue inside your issue stream where each event has an associated replay to help debug the root cause.
```

inconsistent casing with "issue stream"